scrapers/CherryPimps.yml

```diff
@@ -46,7 +46,7 @@ xPathScrapers:
           - parseDate: January 2, 2006
       Performers: &perfAttr
         Name: //div[@class='update-info-block models-list-thumbs']//span/text()
-      Details: &detailsSel //h4[normalize-space('Description:')]/following-sibling::p/text()
```

Comment:
normalize-space with no spaces surrounding it is useless since it collapses spaces. contains is more lenient